lib/openmic.rb

```diff
@@ -0,0 +1,28 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(hash)
+    @location = hash[:location]
+    @date = hash[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
+    # true if user has told joke (@user.tell) to another user
+    # check if same joke is in both users joke array?
+    # ^ But what if they've both learned the joke but not told it
+    # keep track of told jokes?
+
+    # if @ali.told_jokes == [] || @sal.told_jokes == []
```

Comment:
This will return true if either ali or sal have told jokes.  I like that you are storing their told jokes, but it might be easier to store told_jokes on the OpenMic itself and then use the Array method `.uniq` to see help you see if there are any duplicate values.